### PR TITLE
Add mistral large sut hosted on vertex ai

### PR DESCRIPTION
**Pull request by rogthefrog**
[Story](https://github.com/mlcommons/modelbench/issues/607)

### CODE REVIEW

File: plugins/mistralai_sut/modelgauge/suts/vertexai_client.py
Comment (wpietri) [inline plugins/mistralai_sut/modelgauge/suts/vertexai_client.py:81]: Any production SUT will need to have some retry logic sufficient to overcome transient errors and to keep retrying on rate limits.

**Review by wpietri — CHANGES_REQUESTED** [review]

File: plugins/mistralai_sut/modelgauge/suts/vertexai_client.py
Comment (rogthefrog) [reply plugins/mistralai_sut/modelgauge/suts/vertexai_client.py:81]: OK

File: plugins/mistralai_sut/modelgauge/suts/vertexai_client.py
Comment (rogthefrog) [reply plugins/mistralai_sut/modelgauge/suts/vertexai_client.py:43]: OK

File: plugins/mistralai_sut/modelgauge/suts/vertexai_client.py
Comment (rogthefrog) [reply plugins/mistralai_sut/modelgauge/suts/vertexai_client.py]: I've added retry logic to the MistralAI sut, and left a comment to revisit this client if/when we use VertexAI again in the future.

**Comment by rogthefrog** [comment]
@wpietri @bkorycki could you please take another look, and approve if it's OK (to remove "merging is blocked")?

**Comment by wpietri** [comment]
> could you please take another look, and approve if it's OK (to remove "merging is blocked")?

I have approved, so I think it's down to @bkorycki  now.

File: plugins/vertexai/modelgauge/suts/vertexai_client.py
Comment (bkorycki) [inline plugins/vertexai/modelgauge/suts/vertexai_client.py:66]: How does this work? Does it require you to have credentials setup in gcloud CLI?

File: plugins/mistral/modelgauge/suts/mistral_sut.py
Comment (bkorycki) [inline plugins/mistral/modelgauge/suts/mistral_sut.py:86]: The version should be explicit in the uid e.g. `mistral-large-2411`

File: plugins/vertexai/modelgauge/suts/vertexai_mistral_sut.py
Comment (bkorycki) [inline plugins/vertexai/modelgauge/suts/vertexai_mistral_sut.py:108]: I think @bollacker has mentioned in the past that the provider should also be a part of the UID. Especially if we supply multiple providers for one SUT.

**Review by bkorycki — CHANGES_REQUESTED** [review]

**Comment by bkorycki** [comment]
I get an error when I run `poetry run modelgauge run-sut --sut mistral-large-latest`:
```
pydantic_core._pydantic_core.ValidationError: 1 validation error for TextPrompt
text
  Input should be a valid string [type=string_type, input_value=None, input_type=NoneType]
```

File: plugins/vertexai/modelgauge/suts/vertexai_client.py
Comment (rogthefrog) [reply plugins/vertexai/modelgauge/suts/vertexai_client.py:66]: You don't need the gcloud CLI itself, but you do need the application default credentials. I'll add that to the documentation.

**Review by bkorycki — APPROVED** [review]
I'm not sure that the authentication in the vertex AI plugin works the way we would want. But I don't want to hold up merging this PR since we won't actually be using that plugin for any upcoming runs, so we should just revisit that later on.

**Pull request merged**
